secret-store/godelegator-secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: godelegator-secrets
+spec:
+  refreshInterval: "15s"
+  secretStoreRef:
+    name: vault-backend
+    kind: SecretStore
+  data:
+    - secretKey: DATABASE_URL
```

Comment:
Is the `DATABASE_URL` defined as public address? If not, it would be okay/better to not fetch it from vault secrets.

Reply:
sorry, what do you mean by public address? it contains pw for the db so I thought it was necessary

Reply:
I meant public IP 🍭  Aha, it is DB connection string, not just URL! Got it! Thank you!